packages/internal-playwright-helpers/src/pages/cognito.ts

```diff
@@ -33,13 +33,13 @@ export class CognitoPage {
   }
 
   async login(username: string, password: string) {
-    await this.page.fill(".visible-lg [type=text]", username);
-    await this.page.fill(".visible-lg [type=password]", password);
+    await this.page.getByRole('textbox', { name: 'Username' }).fill(username);
+    await this.page.getByRole('textbox', { name: 'Password' }).fill(password);
     await Promise.all([
-      // It is important to call waitForNavigation before click to set up waiting.
-      this.page.waitForNavigation(),
-      // Clicking the link will indirectly cause a navigation.
-      this.page.click(".visible-lg [aria-label=submit]"),
+        // It is important to call waitForURL before click to set up waiting.
+        this.page.waitForURL("https://login.inrupt.com/openid"),
```

Comment:
isnt this too specific since we have have other IDP domains for other envs than podspaces that this is used with?